### ISSUE TITLE
Looks like G Suite setup docs are not formatted properly

### ISSUE THREAD

**Issue by CoreyGriffin** [google/loaner#66]
### Expected Behaviour

Markdown table of setting adjustments to make for your G Suite environment is not formatted correctly

### Actual Behaviour

A bunch of text garbled together, looks like [776a486](https://github.com/google/loaner/commit/776a486639309f5cffbb778d622e8daf2228fc96#diff-16e561a6515f552442a9ad62cfe99382) might have effected this page

### Steps to Reproduce

Go to https://github.com/google/loaner/blob/master/docs/gngsetup_part4.md

**Comment by mikepsada** [comment]
Is anyone from Google monitoring this Github?

**Comment by helfrichmichael** [comment]
@mikepsada Yep! Sometimes issues slip through the cracks. Unfortunately it looks like the markdown flavor didn't match up with what we expected. I'll file an internal bug to track this.

**Comment by helfrichmichael** [comment]
@mikepsada Yep! Sometimes issues slip through the cracks. Unfortunately it looks like the markdown flavor didn't match up with what we expected. I'll file an internal bug to track this.

**Comment by helfrichmichael** [comment]
@mikepsada Yep! Sometimes issues slip through the cracks. Unfortunately it looks like the markdown flavor didn't match up with what we expected. I'll file an internal bug to track this.
